### ISSUE TITLE
How can I use Solarized Light Theme while I am using set background=dark in myvimrc.

### ISSUE THREAD

**Issue by shivanshubisht** [itchyny/lightline.vim#548]
I am using set background=dark because changing it to light interferes with the color of the text displayed on vim.

**Comment by yhu266** [comment]
Could you share your configuration files?

**Comment by shivanshubisht** [comment]
I am currently using nvim through windows terminal and I have edited its One Half light color theme through settings.json file.

    "schemes": [
{
            "name": "One Half Light",
            "background": "#002b36",
            "black": "#282C34",
            "blue": "#61AFEF",
            "brightBlack": "#5A6374",
            "brightBlue": "#61AFEF",
            "brightCyan": "#56B6C2",
            "brightGreen": "#98C379",
            "brightPurple": "#87FFAF",
            "brightRed": "#E06C75",
            "brightWhite": "#DCDFE4",
            "brightYellow": "#FFFF66",
            "cursorColor": "#FFFFFF",
            "cyan": "#56B6C2",
            "foreground": "#DCDFE4",
            "green": "#98C379",
            "purple": "#C678DD",
            "red": "#E06C75",
            "selectionBackground": "#FFFFFF",
            "white": "#DCDFE4",
            "yellow": "#E5C07B"
	},
]

**Comment by itchyny** [comment]
This is not supported but there is a workaround.
```vim
set background=light
execute 'source' globpath(&rtp, 'autoload/lightline/colorscheme/solarized.vim')
set background=dark
```
The light/dark variants were merged in #175 (a9fc5fc62ea5c44b6329ecca0fc1a23ac964968f) which aims at improved compatibility with the colorscheme and code deduplication. It's unlikely to change this back so use the above workaround.
